bump_version

```diff
@@ -54,37 +50,28 @@ else
     new_version_deb="${new_version}-1"
 fi
 
-# Set the Dockerfile web version to the specified new_version
-old_version="$(
-    grep "JELLYFIN_WEB_VERSION=" Dockerfile \
-        | sed -E 's/ARG JELLYFIN_WEB_VERSION=v([0-9\.]+[-a-z0-9]*)/\1/'
-)"
-echo $old_version
-
-old_version_sed="$( sed 's/\./\\./g' <<<"${old_version}" )" # Escape the '.' chars
-sed -i "s/${old_version_sed}/${new_version}/g" Dockerfile*
+# Update the metapackage equivs file
+debian_equivs_file="debian/metapackage/jellyfin"
+sed -i "s/${old_version_sed}/${new_version}/g" ${debian_equivs_file}
```

Comment:
For right now, it's just the deb changelog and fedora changelogs that apply here. It seems I accidentally removed Docker though, which I'll re-add.

Reply:
Oh right, the reason is that I plan to update the Dockerfiles in a separate PR to use the split + meta format as well, so I just removed it from bump_version for now to avoid cluttering this PR further.